### PR TITLE
Ubstrim

### PR DESCRIPTION
**Pull request by GuyonBroadway**
Fug :DDDDDDDD

ALL IS GOOD, WE CAN GO GO GO.

fixes https://github.com/HippieStation/HippieStation/pull/5671
fixes https://github.com/HippieStation/HippieStation/pull/5670
fixes https://github.com/HippieStation/HippieStation/pull/5669
fixes https://github.com/HippieStation/HippieStation/pull/5668
fixes https://github.com/HippieStation/HippieStation/pull/5667
fixes https://github.com/HippieStation/HippieStation/pull/5666
fixes https://github.com/HippieStation/HippieStation/pull/5665
fixes https://github.com/HippieStation/HippieStation/pull/5664
fixes https://github.com/HippieStation/HippieStation/pull/5663
fixes https://github.com/HippieStation/HippieStation/pull/5662
fixes https://github.com/HippieStation/HippieStation/pull/5661
fixes https://github.com/HippieStation/HippieStation/pull/5660
fixes https://github.com/HippieStation/HippieStation/pull/5659
fixes https://github.com/HippieStation/HippieStation/pull/5658
fixes https://github.com/HippieStation/HippieStation/pull/5657
fixes https://github.com/HippieStation/HippieStation/pull/5655
fixes https://github.com/HippieStation/HippieStation/pull/5654
fixes https://github.com/HippieStation/HippieStation/pull/5652
fixes https://github.com/HippieStation/HippieStation/pull/5651
fixes https://github.com/HippieStation/HippieStation/pull/5650
fixes https://github.com/HippieStation/HippieStation/pull/5649
fixes https://github.com/HippieStation/HippieStation/pull/5648
fixes https://github.com/HippieStation/HippieStation/pull/5647
fixes https://github.com/HippieStation/HippieStation/pull/5646
fixes https://github.com/HippieStation/HippieStation/pull/5645
fixes https://github.com/HippieStation/HippieStation/pull/5643
fixes https://github.com/HippieStation/HippieStation/pull/5642
fixes https://github.com/HippieStation/HippieStation/pull/5641
fixes https://github.com/HippieStation/HippieStation/pull/5640
fixes https://github.com/HippieStation/HippieStation/pull/5639
fixes https://github.com/HippieStation/HippieStation/pull/5638
fixes https://github.com/HippieStation/HippieStation/pull/5637
fixes https://github.com/HippieStation/HippieStation/pull/5636
fixes https://github.com/HippieStation/HippieStation/pull/5635
fixes https://github.com/HippieStation/HippieStation/pull/5634
fixes https://github.com/HippieStation/HippieStation/pull/5632
fixes https://github.com/HippieStation/HippieStation/pull/5630
fixes https://github.com/HippieStation/HippieStation/pull/5628
fixes https://github.com/HippieStation/HippieStation/pull/5627
fixes https://github.com/HippieStation/HippieStation/pull/5626
fixes https://github.com/HippieStation/HippieStation/pull/5625
fixes https://github.com/HippieStation/HippieStation/pull/5624
fixes https://github.com/HippieStation/HippieStation/pull/5623
fixes https://github.com/HippieStation/HippieStation/pull/5622
fixes https://github.com/HippieStation/HippieStation/pull/5620
fixes https://github.com/HippieStation/HippieStation/pull/5619
fixes https://github.com/HippieStation/HippieStation/pull/5618
fixes https://github.com/HippieStation/HippieStation/pull/5616
fixes https://github.com/HippieStation/HippieStation/pull/5615
fixes https://github.com/HippieStation/HippieStation/pull/5610
fixes https://github.com/HippieStation/HippieStation/pull/5608
fixes https://github.com/HippieStation/HippieStation/pull/5607
fixes https://github.com/HippieStation/HippieStation/pull/5606
fixes https://github.com/HippieStation/HippieStation/pull/5605
fixes https://github.com/HippieStation/HippieStation/pull/5604
fixes https://github.com/HippieStation/HippieStation/pull/5596
fixes https://github.com/HippieStation/HippieStation/pull/5578
fixes https://github.com/HippieStation/HippieStation/pull/5576
fixes https://github.com/HippieStation/HippieStation/pull/5574
fixes https://github.com/HippieStation/HippieStation/pull/5565
fixes https://github.com/HippieStation/HippieStation/pull/5564
fixes https://github.com/HippieStation/HippieStation/pull/5562
fixes https://github.com/HippieStation/HippieStation/pull/5558
fixes https://github.com/HippieStation/HippieStation/pull/5557
fixes https://github.com/HippieStation/HippieStation/pull/5555
fixes https://github.com/HippieStation/HippieStation/pull/5554
fixes https://github.com/HippieStation/HippieStation/pull/5553
fixes https://github.com/HippieStation/HippieStation/pull/5551
fixes https://github.com/HippieStation/HippieStation/pull/5550
fixes https://github.com/HippieStation/HippieStation/pull/5549
fixes https://github.com/HippieStation/HippieStation/pull/5548
fixes https://github.com/HippieStation/HippieStation/pull/5547
fixes https://github.com/HippieStation/HippieStation/pull/5545
fixes https://github.com/HippieStation/HippieStation/pull/5544
fixes https://github.com/HippieStation/HippieStation/pull/5542
fixes https://github.com/HippieStation/HippieStation/pull/5541
fixes https://github.com/HippieStation/HippieStation/pull/5539
fixes https://github.com/HippieStation/HippieStation/pull/5538
fixes https://github.com/HippieStation/HippieStation/pull/5535
fixes https://github.com/HippieStation/HippieStation/pull/5534
fixes https://github.com/HippieStation/HippieStation/pull/5533
fixes https://github.com/HippieStation/HippieStation/pull/5532

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,3 +1,66 @@
 # This list auto requests reviews from the specified org members
 # when a PR that modifies the file in question is opened
 # This list is alphabetized by User -> Filename KEEP IT THAT WAY
+
```
Comment (ghost) [inline .github/CODEOWNERS]: Remove all of these please

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
-[Directions]: # (If you discovered this issue from playing HippieStation hosted servers
+[Round ID]: # (If you discovered this issue from playing tgstation hosted servers:)
```
Comment (ghost) [inline .github/ISSUE_TEMPLATE.md]: tg?

File: .github/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -1,9 +1,9 @@
...
+[Round ID]: # (**INCLUDE THE ROUND ID**)
+[Round ID]: # (It can be found in the Status panel or retrieved from https://atlantaned.space/statbus/round.php ! The round id let's us look up valuable information and logs for the round the bug happened.)
```
Comment (ghost) [inline .github/ISSUE_TEMPLATE.md]: wrong url

**Review by ghost — DISMISSED** [review]

**Comment by GuyonBroadway** [comment]
lets try the travis trick again.

**Pull request closed without merging**